### PR TITLE
README: Add mission statement

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
Relates to https://github.com/ansible-collections/community.postgresql/discussions/544

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,19 @@
 
 This collection is a part of the Ansible package.
 
+## Our mission
+
+At `community.postgresql` Ansible collection project,
```
Comment (hunleyd) [inline README.md:10]:
```suggestion
At the `community.postgresql` Ansible collection project,
```

File: README.md [hunk 2]
```diff
@@ -5,6 +5,19 @@
...
+our mission is to produce and maintain simple, flexible,
+and powerful open-source software tailored to automating PostgreSQL-related tasks.
+
+We welcome members from all skill levels to participate actively in our open, inclusive, and vibrant community.
+Whether you are a expert or just beginning your journey with Ansible and PostgreSQL,
```
Comment (hunleyd) [inline README.md:15]:
```suggestion
Whether you are an expert or just beginning your journey with Ansible and PostgreSQL,
```

Comment (Andersson007) [reply README.md:15]: ah, removed "seasoned" but forgot to change the article.. thanks for spotting!

File: README.md (hunk 1 above)
Comment (Andersson007) [reply README.md]: good catch, thanks, PTAL

**Review by hunleyd — APPROVED** [review]

**Pull request merged**